anvil/Cargo.toml

```diff
@@ -23,7 +23,7 @@ profiling = { version = "1.0" }
 
 [dependencies.smithay]
 default-features = false
-features = ["desktop", "wayland_frontend"]
+features = ["desktop", "wayland_frontend", "use_bindgen"]
```

Comment:
I don't have anything against this feature, but can we please not enable it for anvil by default. None of our default platforms need it.